src/docs/product/performance/index.mdx

```diff
@@ -26,6 +26,12 @@ Using the information on this page, you can trace issues back through services (
 
 ## Filter Performance Data
 
+<Alert level="info" title="Important">
+
+For organizations participating in the limited release, the transaction filter on the Performance page has been simplified to support metric enhanced performance. The search bar can only be used to filter by transaction name. See [transaction search](/product/performance/filters-display/#transaction-search).
+
+</Alert>
+
```

Comment:
Might be good to clarify the limited release, and to explain the difference from other searches! Maybe something like

"The transaction filter has been simplified for organizations participating in the limited release of Metrics Enhanced Performance. See blah blah"

Then the "transaction search" section can go into more detail

Reply:
Yeah I do agree, I was aiming to be consistent with the following documentation, as we have a standard blurb before any beta feature
https://sentry.slack.com/archives/C02N6582ZFA/p1659603015550699?thread_ts=1659573427.357199&cid=C02N6582ZFA

I feel like we'd have to update the blurb everywhere.

Reply:
I'm confused, the banner there is different from yours, what's being kept consistent? Also, the banner there is under the "Server-side sampling" heading, which is indeed in limited release. Here, the alert implies that "Filter Performance Data" is in limited release which is ... kind of the case? Maybe PM should tiebreak this one

Reply:
I see the confusion, so this alert I meant to put an alert I wanted to state that this doesn't really apply if your in the MEP release. And to click the link to see the updated functionality for transaction name search.